src/DNSC/Iterative.hs

```diff
@@ -144,40 +173,108 @@ runIterative cxt sa n = withNormalized n (iterative sa) cxt
 
 ---
 
+replyMessage :: Either QueryError [ResourceRecord]
```

Comment:
`Either QueryError [ResourceRecord]` is the search result, and The `Identifier` and `Question` are arguments derived from the request.
I did this because I thought it would be easier to understand if ones from the request  were next to each other.